2018-01-03 10:19:09 TRACE - gwt account method invoked
2018-01-03 10:19:09 DEBUG - checking the aid against the map key
2018-01-03 10:19:09 INFO - account ID 1002 found
2018-01-03 10:19:09 DEBUG - to account retrieved
2018-01-03 10:19:09 INFO - 1001 and 1002 accounts are updates successfully
2018-01-03 10:19:09 INFO - successfully transfered
2018-01-03 10:19:09 INFO - transfer the amount rs 5000 from account ID 1003 to 1002
2018-01-03 10:19:09 TRACE - transferfund() invoked in service
2018-01-03 10:19:09 TRACE - gwt account method invoked
2018-01-03 10:19:09 DEBUG - checking the aid against the map key
2018-01-03 10:19:09 INFO - account ID 1003 found
2018-01-03 10:19:09 DEBUG - from account retrieved
2018-01-03 10:19:09 TRACE - gwt account method invoked
2018-01-03 10:19:09 DEBUG - checking the aid against the map key
2018-01-03 10:19:09 INFO - account ID 1002 found
2018-01-03 10:19:09 DEBUG - to account retrieved
2018-01-03 10:19:09 ERROR - no sufficient fund to transfer 
com.sapient.exceptions.BalException: no sufficient fund to transfer 
	at com.sapient.service.SerImpl.transferFund(SerImpl.java:41)
	at com.sapient.client.TransferClient.main(TransferClient.java:31)
2018-01-03 10:19:09 INFO - transfer the amount rs 5000 from account ID 2003 to 1002
2018-01-03 10:19:09 TRACE - transferfund() invoked in service
2018-01-03 10:19:09 TRACE - gwt account method invoked
2018-01-03 10:19:09 DEBUG - checking the aid against the map key
2018-01-03 10:19:09 ERROR - no id found
com.sapient.exceptions.IdException: no id found
	at com.sapient.dao.DaoImpl.getAccount(DaoImpl.java:31)
	at com.sapient.service.SerImpl.transferFund(SerImpl.java:35)
	at com.sapient.client.TransferClient.main(TransferClient.java:39)
